myapp/app/controllers/tasks_controller.rb

```diff
@@ -3,9 +3,12 @@
 class TasksController < ApplicationController
   before_action :logged_in_user, only: [:edit, :update, :destroy]
 
-  def index
+  def index # rubocop:disable Metrics/AbcSize
     user_id = session[:user_id]
     @tasks = Task.search(user_id: user_id, status: params[:status], keyword: params[:keyword], sort: params[:sort], direction: params[:direction]).page(params[:page])
+    @tasks = @tasks.includes(:labels)
+    @tasks = @tasks.includes(:task_labels)
```

Comment:
（任意です）
一つにまとめられそうですね

```suggestion
    @tasks = @tasks.includes(:labels, :task_labels)
```